platform/qt/src/sqlite3.cpp

```diff
@@ -60,20 +60,23 @@ void checkDatabaseOpenError(const QSqlDatabase &db) {
     }
 }
 
+namespace {
+    thread_local uint64_t count = 0;
```

Comment:
We cannot use `thread_local` because it doesn't work on iOS. Use `mbgl::util::ThreadLocal` instead.

Reply:
https://github.com/mapbox/mapbox-gl-native/pull/9448